// Try to open a file (simulate with a message), catch an error, and always log "Done!" in finally


try{
    let transfer = abc
}
catch(error){
    console.log('Insufficient balance');
}
finally{
    console.log('Thank you for using our service');
}